### DISCUSSION TITLE
Allow "Date is in the past" warning to be optional in the time node

### DISCUSSION THREAD

**Discussion by AndreiArdelean1** [zachowj/node-red-contrib-home-assistant-websocket#1228] [Feature Request]
### Describe the bug

The "time node" throws a "Date is in the past" warning when the entity's date is in the past and "Repeat daily" is off. Instead of the warning maybe an option "Trigger if the date is in the past" can be added. This way the debug console doesn't get cluttered and the user can choose how to handle dates that are in the past.

### To Reproduce

1. Create a time node 
2. Set "Repeat daily" to off
3. Deploy the changes
4. Change the entity's date to a date in the past

### Environment Information

Version: 0.62.2

Home Assistant version: 2023.12.4
Companion version:

Node-RED version: 3.1.3
Docker: yes
Add-on: no

Node.js version:
OS: Ubuntu

**Comment by WhiteSockedDancer** [comment]
I got the same warning. Maybe there should be an option to disable the "Date is in the past" warnings?

If my "time" item property  is cleared i also get the error, that the string is no time. i also would like to disable that warning since i reset the time-attribute sometimes. (i could also set a time in the past if i could disable these warnings.

**Comment by zachowj** [comment]
Added a1e16ee6f16bb4701165477be2b649ce27eb4620
